entries/a/au.com.json

```diff
@@ -0,0 +1,19 @@
+{
+  "au": {
+    "domain": "au.com",
+    "url": "https://www.au.com/",
```

Comment:
The `url` tag isn't needed if the domain redirects to the same page as the value of the `url` tag.
```suggestion
```